packages/web/src/services/WebWorker.js

```diff
@@ -7,20 +7,21 @@ export default class WebWorker {
    * Initializes a web worker for performing async tasks.
    * @param {file} workerFile The web worker code, which is turned into a string an exec'd.
    * @param {?boolean} terminateOnResult Whether or not to terminate the worker on gathering the result.
+   * @param {?Array<file>} dependencies Optional array of file dependencies they worker needs
```

Comment:
I think the answer is no, but just wanted to confirm this can't be used as an exploit? As in, we don't pass any raw user input to this argument?

Reply:
we don't have any UGC here. good call out though.